File: deletion.java
// Starting at the index 1, we shift each element one position to the left
for (int i = 1; i < length; i++){
  // Shift each element one position to the left
  int_array[i - 1] = int_array[i];
}

length --;

// Say we want to delete the element at index 1
for (int i = 2; i < length; i++){
  // Shift each element one positio to the left
  int_array[i - 1] = int_array[i];
}

// The length needs to be consisitent with the current 
length --;

File: deletion.py
# Initialize an integer array with some elements
int_array = [1, 2, 3, 4, 5]
length = len(int_array)

# Starting at index 1, shift each element one position to the left
for i in range(1, length):
  # Shift each element one position to the left
  int_array[i - 1] = int_array[i]

# Decrease the lenght
length -= 1

print(int_array)

# Remove the last element since it's now duplicated
int_array.pop()
print(int_array)